.gitignore

```diff
@@ -83,6 +84,8 @@ samples/web/Continote/public/scripts/config.js
 # Libraries for samples
 samples/android/Continote/app/libs/*.aar
 samples/chrome-extension/Continote/scripts/firebase-continue.js
+samples/ios/Continote/Continote/FCNContinue.h
```

Comment:
Yes, we did talk about that. I actually decided to go the route of copy-pasting to match the installation guide for any app a developer makes, so it guides them through the full process even just for this sample.

Do you think we should use symlinks instead? That would shorten the setup process by a short step or two, but then it would deviate from the installation guide of the library itself (for any other app to use the library, that is).